schemas/wall/dd_wall.xsd

```diff
@@ -2071,6 +2071,7 @@
 					<xs:documentation>Parameters of the BRDF model for various wall components (grid subsets)</xs:documentation>
 					<xs:appinfo>
 						<coordinate1>1...N</coordinate1>
+						<units>mixed</units>
```

Comment:
This is correct since the units depend on the particular BRDF model used but it appears the units aren't specified anywhere.  I think we should add them to `wall_brdf_identifier.xml` for each model.  We should do this everywhere we allow `mixed` units - but not necessarily in this PR!